ops/jenkins/bfd-run-sonarqube/Jenkinsfile

```diff
@@ -37,10 +37,10 @@ spec:
       mountPath: /var/run
     resources:
       requests:
-        memory: '8192Mi'
+        memory: '16384Mi'
         cpu: '4000m'
       limits:
-        memory: '8192Mi'
+        memory: '16384Mi'
```

Comment:
This aligns the resources for the test container infrastructure with the[ sizing for hosted github actions runners](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) where resource exhaustion hasn't historically been a problem.